troposphere/static/js/components/common/InteractiveDateField.react.js

```diff
@@ -48,13 +48,31 @@ define(function (require) {
 
     render: function () {
       var labelEl = this.props.labelText ? (<label>{this.props.labelText}</label>) : "";
+      var style = {};
+      style.button = {cursor: "pointer"};
       return (
-        <div className="form-group">
-          {labelEl}
-          <input type='text' className='form-control' value={this.state.value} onChange={this.onValueChanged}/>
-          <span className="input-group-addon" id="enddate-set-addon" onClick={this.setEndDateNow}>Today</span>
-          <span className="input-group-addon" id="enddate-clear-addon" onClick={this.unsetDate}>Clear</span>
-        </div>
+            <div className="form-group">
+                    <label htmlFor="date-field">{labelEl}</label>
+                <div className="input-group">
+                    {/* Button, !button because bootstrap */}
```

Comment:
LOL This is just a bad comment. I thought I was being terse and clever.
Just to say the div was being used as a button, it's not an actual button because of the wonderful world of bootstrap.